js/src/pages/edit-paid-ads-campaign/index.js

```diff
@@ -55,18 +79,72 @@ const EditPaidAdsCampaign = () => {
 		);
 	}
 
+	const handleSubmit = async ( values, submitter ) => {
+		const { action } = submitter.dataset;
+		const { amount } = values;
+
+		try {
+			await updateAdsCampaign( campaign.id, { amount } );
+
+			if ( action === ACTION_COMPLETE ) {
```

Comment:
💅  I didn't mention it last review since I had doubts. But reading it again... I think could be better to rename this to `ACTION_SAVE_ASSETS` to be clearer about what we are doing.

Reply:
I struggled with this naming while working on this PR. To be precise, the action `ACTION_COMPLETE` refers to saving the campaign and its assets, and the action `ACTION_SKIP_ASSETS` refers to saving the campaign only. So ideally it would be
`ACTION_SUBMIT_CAMPAIGN_AND_ASSETS` and `ACTION_SUBMIT_CAMPAIGN_ONLY`.

But this makes the action naming look inconsistent with the text of the associated buttons, so in the end I chose to match the button texts.

Would you prefer the naming of `ACTION_SUBMIT_CAMPAIGN_AND_ASSETS` and `ACTION_SUBMIT_CAMPAIGN_ONLY`? Or other names?

Reply:
Not a strong opinion if it's clear what is doing. So I maybe the two above, or the ones you mentioned also good 👌 

`ACTION_SUBMIT_CAMPAIGN_WITH_ASSETS`
`ACTION_SUBMIT_CAMPAIGN_SKIP_ASSETS`

Reply:
Renamed in https://github.com/woocommerce/google-listings-and-ads/pull/1799/commits/56af208bb36ac149aa89e19b1c55345b1fac3150.